src/en/flamecomics/src/eu/kanade/tachiyomi/extension/en/flamecomics/FlameComics.kt

```diff
@@ -4,62 +4,158 @@ import android.graphics.Bitmap
 import android.graphics.BitmapFactory
 import android.graphics.Canvas
 import android.graphics.Rect
-import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.GET
 import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
 import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.decodeFromString
 import okhttp3.Interceptor
 import okhttp3.MediaType.Companion.toMediaType
 import okhttp3.Protocol
+import okhttp3.Request
 import okhttp3.Response
 import okhttp3.ResponseBody.Companion.toResponseBody
 import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
 import java.io.ByteArrayOutputStream
+import java.net.URLDecoder
 
-class FlameComics : MangaThemesia(
-    "Flame Comics",
-    "https://flamecomics.xyz",
-    "en",
-    mangaUrlDirectory = "/series",
-) {
+class FlameComics : ParsedHttpSource() {
+    override val name = "Flame Comics"
+    private val host = "flamecomics.xyz"
+    override val baseUrl = "https://$host"
```

Comment:
It should be fine to repeat the `host` here and in `cdn`. Semantically you're saying that the CDN must be under the main domain, which isn't always the case.